### ISSUE TITLE
A bug in explicit-member-accessibility when a member start with "#"

### ISSUE THREAD

**Issue by HypnosNova** [typescript-eslint/typescript-eslint#2339]
**Repro**

```JSON
{
    rules: {
        '@typescript-eslint/explicit-member-accessibility': 'error'
    }
};
```

```TS
class A {
    #aaa() {}
}
```

**Expected Result**
no error

**Actual Result**
shows that I musr add **public** or **private** before the method aaa()

The # means the member is private so we can't use **public** or **private** in this case.

**Additional Info**

**Versions**

| package                                   | version |
| ----------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin-tslint` | `3.1.0` |
| `@typescript-eslint/eslint-plugin`               | `3.7.1` |
| `@typescript-eslint/parser`               | `3.7.1` |
| `TypeScript`                              | `3.9.6` |
| `ESLint`                                  | `7.5.0` |
| `node`                                    | `12.16.1` |
| `npm`                                     | `6.13.4` |

**Comment by bradzacher** [comment]
We do not fully support private class members yet.
https://github.com/typescript-eslint/typescript-eslint/issues/1436
